docs/interfaces.html

```diff
@@ -0,0 +1,272 @@
+    <h2>Interface</h2>
+
+    <p>
+        Interface functions are abstract, they take wallet contents and options
+        as input, and they produce wallet contents and side effects as output.
+    </p>
+
+    <p>
+        In order to ensure portability, wallet interfaces MUST be functions of
+        JSON, and not take complex instances of classes, functions or other
+        types of object.
+    </p>
+
+    <p>
+        A wallet implementation MAY implement a subset of the interfaces defined below.
+
+        For example:
+    <ul>
+        <li>A verifiable credential wallet might not implement the <code>Transfer</code> interface.
+        Also, the <code>Query</code> interface of a verifiable credential wallet might be specific to credential search.</li>
+        <li>Similarly, a wallet implementation which relies on an external storage, such as encrypted data vaults, might choose not to implement <code>Export</code>, <code>Import</code>, <code>Lock</code>, and/or <code>Unlock</code> interfaces.</li>
+    </ul>
+
+    </p>
+
+    <section>
+        <h3>Import</h3>
+        <p>
+        Takes a serialized exported wallet representation as input. Loads the
+        representation into wallet software.
+        </p>
+        <p>
+        This method may not be necessary for implementations that rely on
+        external storage, such as Encrypted Data Vaults.
+        </p>
+    </section>
+
+    <section>
+        <h3>Export</h3>
+
+        <p>Only ciphertext wallet contents can be exported.</p>
+
+        <p>Produces a serialized exported wallet representation.</p>
+
+        <p>
+            This method may not be necessary for implementations that rely on
+            external storage, such as Encrypted Data Vaults.
+        </p>
+    </section>
+
+
+    <section>
+        <h3>Unlock</h3>
+        <p>
+        Transforms one or more wallet contents from ciphertext to plaintext.
+        </p>
+
+        <p>Requires knowledge of the password used to lock the wallet.</p>
+
+        <p>
+        This method may not be necessary for implementations that rely on
+        external storage, such as Encrypted Data Vaults.
+        </p>
+    </section>
+
+    <section>
+        <h3>Lock</h3>
+        <p>
+        Transforms one or more wallet contents from plaintext to ciphertext.
+        </p>
+        <p>Requires knowledge of the password used to lock the wallet.</p>
+
+        <p>
+        This method may not be necessary for implementations that rely on
+        external storage, such as Encrypted Data Vaults.
+        </p>
+
+        <div class="issue" data-number="5">
+        Mostly a reference implementation issue. Locked representation should
+        indicate a best practice.
+        </div>
+    </section>
+
+    <section>
+        <h3>SignRaw</h3>
+        <p>
+        Takes a Buffer as input, and an <code>options</code> object, which
+        contains at least a <code>verificationMethod</code>, and may content
```

Comment:
not sure about this...
```suggestion
        contains at least a <code>verificationMethod</code>, and may contain
```